com_redhat_kdump/service/kdump.py

```diff
@@ -85,6 +86,11 @@ def reserved_memory(self):
 
     @reserved_memory.setter
     def reserved_memory(self, value):
+        if value != "auto":
+            if int(value) > self._upper:
+                value = str(int(self._upper))
+            if int(value) < self._lower:
+                value = str(int(self._lower))
```

Comment:
Maybe you can put the above code inside a standalone function and unit-test this function (choose three values, one < lowerBound, one between lowerBound and upperBound and one  > upperBound) instead? I think this shall make the test easier and there is also no need to change test/unit_tests/test_kickstart.py.

Reply:
different plat or different machine with test the “_lower“ ”_upper” will have different value. the set value oftest/unit_tests/test_kickstart.py  must  between  the “_lower“ ”_upper” ， this is the pr we want to do